conf/nginx-nr-agent.ini

```diff
@@ -0,0 +1,46 @@
+# global settings
```

Comment:
this is the nginx-nr-agent.ini configuration to be used to run the nginx-nr-agent